src/Traits/PaymentSources/PointsTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\PhpSdk\Traits\PaymentSources;
+
+use Symfony\Component\Serializer\Annotation\Groups;
+
+trait PointsTrait
```

Comment:
@ketanp77 Looking at the requests sent by Jose in his tests, I can't see `externalId`, and we're missing `api_key` and `points`.

```json
"payment_source": {
    "account_id": "<account_id>",
    "provider_id": "<provider_id>",
    "api_key": "<api_key>",
    "points": 46,
    "type": "points"
  },
```

Reply:
He didn't send me those ...I will add 'em

Reply:
Fixed.